### DISCUSSION TITLE
How do I limit which table a user can view ?

### DISCUSSION THREAD

**Discussion by dittorahmat** [nocodb/nocodb#933] [Q&A] [answered]
How do I limit which table a user can view ? Currently what I understand, a user can see all tables in the current project (I assume this is mirroring the database schema), regardless of their role.

Is there a way to limit what tables the user can see on NoCoDB ?

**Comment by dittorahmat** [comment, marked as answer]
I found about UI Access Control here

https://docs.nocodb.com/setup-and-usages/meta-management

But I don't know why my nocodb installation doesn't show this

**Reply by dittorahmat** [reply]
I already got the answer from discord.

Just double click on the import/export meta data tab, and the UI Access Control tab will appear

**Comment by o1lab** [comment]
Converting to an issue
